tools/Makefile.win

```diff
@@ -565,7 +561,7 @@ endif
 # apps_distclean: Perform the distclean operation only in the user application
 #                 directory.
 
-apps_preconfig:
+apps_preconfig: clean_context
```

Comment:
Let' dirlinks depends on clear_context?

Reply:
> Let' dirlinks depends on clear_context?

Wouldn't that mess up context?  It would need some re-organization in it's prerequisites.

Reply:
Yes, the right sequences is:
```
clear_context: .config

include/arch: clear_context
....
dirlinks: include/arch ...
```

the old rule is wrong:
```
dirlinks: clear_context include/arch ...
```
because clear_context and include/arch hasn't relationship in the concurrent build.

Reply:
Now that Liu removed dirlinks from the kconfig targets (which were the primary users of dirlinks + clean_context), is it still an issue?

Reply:
But it isn't good to launch a new instance(same make file) to build dirlinks target.

Reply:
Is this PR going to be updated with these changes?

Reply:
Yes, I'm doing with these changes,  it may need more test and verify locally.